app/views/users/_response_templates.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="crayons-card crayons-card--content-rows">
+<div class="crayons-card crayons-card--content-rows" id="#comment-templates">
```

Comment:
Remove `#` from the id name. This was written with some different context in the original issue.

Reply:
oops! I missed this one during copy paste.